readme.md

```diff
@@ -221,6 +228,33 @@ You can optionally pass one or more of these parameters every time you use Lambo
   ```bash
   lambo new superApplication --full
 
+## GitHub Repository Creation
+
+- `--github=<options>` to create a new GitHub repository and push your new project to it.
+
+  ```bash
+  lambo new superApplication --github='--private'
+  ```
+
+Lambo uses the official GitHub command line tool to create your new repository so you must have it installed. You can find installation instructions [here](https://github.com/cli/cli#installation).
+
+New repository creation is configured by passing the value of the Lambo `--github` command line option directly to the GitHub command line tool. You must pass at least the desired visibility of your new repository, one of, `--private`, `--public`, or `--internal`.
+
+ ```bash
+ # Create a new GitHub repository <your_github_username>/superApplication with the desired visibility
+  lambo new superApplication --github='--private'
+  lambo new superApplication --github='--public'
+  lambo new superApplication --github='--internal'
+ ```
+
+Please refer to the GitHub command line tool [repository creation documentation](https://cli.github.com/manual/gh_repo_create) for the full list of available options.
+
+- `--github-org=ORG` to specify under which organisation the new repository will be created.
+
+ ```bash
+ # Create a new public GitHub repository tighten/superApplication
+ lambo new superApplication --github='--public' --github-org=tighten
```

Comment:
haha love that you answered this here. I wonder if there's an easier syntax for this... hm.

Reply:
What do you think about this API?

![image](https://user-images.githubusercontent.com/1974648/124638675-25ba9900-de83-11eb-8c87-38be970d75e1.png)

So for example:

```
lambo new ozzie --github \
--gh-public \
--gh-description='Keep track of the prs and issues on all your opensource projects' \
--gh-homepage='https://ozzie.tighten.co'
```

Reply:
@mattstauffer ^^

Reply:
I *love* this new idea.